app/components/ui/LandingPageEligibilityBlock.jsx

```diff
@@ -1,8 +1,19 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router';
 import PropTypes from 'prop-types';
+import { eligibilitiesMapping } from '../../utils/refinementMappings'
 
 class LandingPageEligibilityBlock extends Component {
+
+  buildBlockUrl(eligibility) {
+    let url = '/search?'
+    const eligibilityElements = eligibilitiesMapping[eligibility];
+    eligibilityElements.forEach((element, index) => {
+      url += `refinementList[eligibilities][${index}]=${encodeURIComponent(element)}&`;
+    });
+    return url;
+  }
```

Comment:
I think it'd be better to use a library for this, since it's more likely that they'll handle the edge cases when it comes to query parameter escaping. [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) is built in to modern browsers, although if we want to maintain IE11 support, we'll probably want to install a [polyfill](https://www.npmjs.com/package/url-search-params-polyfill).